js/components/utils.js

```diff
@@ -579,11 +579,18 @@ var Utils = {
   * @param {Object} thumbnails - metadata object containing information about thumbnails
   * @param {Number} hoverTime - time value to find thumbnail for
   * @param {Number} duration - duration of the video
+  * @param {Boolean} isVideoVr - if video is vr
   * @returns {Object} object that contains URL and index of requested thumbnail
   */
-  findThumbnail: function(thumbnails, hoverTime, duration) {
+  findThumbnail: function(thumbnails, hoverTime, duration, isVideoVr) {
     var timeSlices = thumbnails.data.available_time_slices;
     var width = thumbnails.data.available_widths[0]; //choosing the lowest size
+    if (isVideoVr && width < 380) {
+      // it is necessary to take bigger image for showing part of the image
+      // so choose not the lowest size but bigger one, the best width is 380
+      var index = thumbnails.data.available_widths.length >= 5 ? 4 : thumbnails.data.available_widths.length - 1;
```

Comment:
How did we choose the value 5 for:

thumbnails.data.available_widths.length >= 5

Reply:
We have 2 sizes for thumbnails width: 80px for small screens and 119px for other. For 360 image we need to get only part of the server image  and show this part. So we need to find the best proportions between thumbnails sizes and images sizes. Usually thumbnailWidth * 4 is enought for show a most viewing part of the image.